orcidio/.htaccess

```diff
@@ -0,0 +1,6 @@
+Options +FollowSymLinks
+RewriteEngine on
+
+# Make ontology artifacts resolvable
+RewriteRule ^orcidio.ofn$ https://raw.githubusercontent.com/cthoyt/orcidio/main/orcidio.ofn [R=302,L]
+RewriteRule ^orcidio.owl$ https://raw.githubusercontent.com/cthoyt/orcidio/main/orcidio.owl [R=302,L]
```

Comment:
I don't think this is what you intent. This would mean that only the URL `https://w3id.org/orcidio/orcidio.owl` works instead of having `https://w3id.org/orcidio` redirect to your ontology directly

Reply:
Hi @dgarijo, thank you for the quick comment. I would like https://w3id.org/orcidio/orcidio.owl to redirect to https://raw.githubusercontent.com/cthoyt/orcidio/main/orcidio.owl (and similarly for the `.ofn` file). I admit there's a bit of redundancy, but I think inside `https://w3id.org/orcidio/` I will add future alternatives like `https://w3id.org/orcidio/orcidio-obo.owl`, `https://w3id.org/orcidio/orcidio-chemistry.owl`. Have I misunderstood your comment?

Reply:
ok